communitydelphi.properties

```diff
@@ -1,15 +1,23 @@
 category=Languages
 description=Code Analyzer for Delphi
 homepageUrl=https://github.com/integrated-application-development/sonar-delphi
-archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0
-publicVersions=1.12.0
+archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0,1.12.0
+publicVersions=1.12.1
 
 defaults.mavenGroupId=au.com.integradev.delphi
 defaults.mavenArtifactId=sonar-delphi-plugin
 
+1.12.1.description=Improvements to PlatformDependentTruncation, bug fixes
+1.12.1.sqs=[10.8,LATEST]
+1.12.1.sqcb=[24.12,LATEST]
+1.12.1.sqVersions=[9.9,10.7]
+1.12.1.date=2024-12-06
+1.12.1.changelogUrl=https://github.com/integrated-application-development/sonar-delphi/releases/tag/v1.12.1
+1.12.1.downloadUrl=https://github.com/integrated-application-development/sonar-delphi/releases/download/v1.12.1/sonar-delphi-plugin-1.12.1.jar
+
 1.12.0.description=Improvements to intrinsics, parsing improvements, bug fixes
-1.12.0.sqs=[10.8,LATEST]
-1.12.0.sqcb=[24.12,LATEST]
+1.12.0.sqs=[10.8.*]
```

Comment:
Unfortunately, asterisks won't work at the beginning of a range. You'll need to make these `[10.8,10.8.8]` &etc.

Reply:
Makes sense. I've updated the version ranges for `1.12.0` and `1.11.0` accordingly.